serialport/serialport.d.ts

```diff
@@ -1,40 +1,17 @@
-// Type definitions for serialport
-// Project: https://github.com/EmergingTechnologyAdvisors/node-serialport 
+// Type definitions for serialport 4.0.1
+// Project: https://github.com/EmergingTechnologyAdvisors/node-serialport
 // Definitions by: Jeremy Foster <https://github.com/codefoster>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped 
-
-declare module 'serialport' {
-    module parsers {
-        function readline(delimiter: string):void;
-        function raw(emitter:any, buffer:string):void
-    }
-
-    export class SerialPort {
-        constructor(path: string, options?: Object, openImmediately?: boolean, callback?: (err:string) => void)
-        isOpen: boolean;
-        on(event: string, callback?: (data?:any) => void):void;
-        open(callback?: () => void):void;
-        write(buffer: any, callback?: (err:string, bytesWritten:number) => void):void
-        pause():void;
-        resume():void;
-        disconnected(err: Error):void;
-        close(callback?: () => void):void;
-        flush(callback?: () => void):void;
-        set(options: setOptions, callback: () => void):void;
-        drain(callback?: () => void):void;
-        update(options: updateOptions, callback?: () => void):void;
-    }
-
-    export function list(callback: (err: string, ports:portConfig[]) => void): void;
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+declare namespace serialport {
```

Comment:
The trouble i'm running in to with `declare module 'serialport'` is that the compiled code when I `import` ends up being like
```TypeScript
var serialport_1 = require('serialport');
new serialport_1.SerialPort()
``` 
which triggers the deprecation warning. I need the `SerialPort` class to be the default from the `require` and also have access to the other interfaces in the TS. Do you have any suggestions for how to accomplish this?